### PR TITLE
C9 c10 jmr edits

### PR DESCRIPTION
**Pull request by jrosen48**
@ivelasq quick Q for you - in the context of c9 (aggregate data), do you have thoughts on mentioning equity and the type of data (district-level)?

i.e., changing the title from:

*Introduction to Aggregate Data*

to:

*Introduction to Aggregate Data to Illuminate Educational Inequities Using District-Level Data*

Perhaps the revision is too wordy. Curious what you think, tho, in general, and possibly mentioning either equity or district-level data if this is too wordy!

### CODE REVIEW

**Comment by jrosen48** [comment]
Hi @ivelasq and @restrellado, only very minor changes to these chapters - I will plan to take a closer look after I've made it through the remaining walkthroughs. @ivelasq, just the one Q above about the title; I think the other changes are minor, but, of course, I welcome your input on those, too.

**Comment by ivelasq** [comment]
I'm fine with any changes! Maybe 'Using School-Level Aggregate Data to Illuminate Educational Inequities' to make a little shorter? (it's district data but school-level)

**Comment by jrosen48** [comment]
like that! will change it

**Comment by jrosen48** [comment]
addressed that in last commit
